src/main/webapp/app/shared/dropdown/CancerTypeSelect.tsx

```diff
@@ -7,44 +7,9955 @@ import { TumorType } from 'app/shared/api/generated/OncoKbPrivateAPI';
+    color: 'LightYellow',
+    mainType: 'Soft Tissue Cancer, NOS',
+    level: 0,
+    tissue: 'Soft Tissue',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 973,
+    subtype: '',
+    code: '',
+    color: 'Purple',
+    mainType: 'Thymic Cancer, NOS',
+    level: 0,
+    tissue: 'Thymus',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 974,
+    subtype: '',
+    code: '',
+    color: 'Purple',
+    mainType: 'Adrenocortical Carcinoma',
+    level: 0,
+    tissue: 'Adrenal Gland',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 975,
+    subtype: '',
+    code: '',
+    color: 'Blue',
+    mainType: 'Pleural Cancer, NOS',
+    level: 0,
+    tissue: 'Pleura',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 976,
+    subtype: '',
+    code: '',
+    color: 'Black',
+    mainType: 'Other Cancer, NOS',
+    level: 0,
+    tissue: 'Other',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 977,
+    subtype: '',
+    code: '',
+    color: 'Orange',
+    mainType: 'Renal Neuroendocrine Tumor',
+    level: 0,
+    tissue: 'Kidney',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 978,
+    subtype: '',
+    code: '',
+    color: 'Blue',
+    mainType: 'Penile Cancer, NOS',
+    level: 0,
+    tissue: 'Penis',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 979,
+    subtype: '',
+    code: '',
+    color: 'HotPink',
+    mainType: 'Breast Cancer, NOS',
+    level: 0,
+    tissue: 'Breast',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 980,
+    subtype: '',
+    code: '',
+    color: 'Orange',
+    mainType: 'Clear Cell Sarcoma of Kidney',
+    level: 0,
+    tissue: 'Kidney',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 981,
+    subtype: '',
+    code: '',
+    color: 'Purple',
+    mainType: 'Vulvar/Vaginal Cancer, NOS',
+    level: 0,
+    tissue: 'Vulva/Vagina',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 982,
+    subtype: '',
+    code: '',
+    color: '',
+    mainType: 'All Tumors',
+    level: -1,
+    tissue: '',
+    tumorForm: 'MIXED',
+  },
+  {
+    id: 983,
+    subtype: '',
+    code: '',
+    color: '',
+    mainType: 'All Liquid Tumors',
+    level: -1,
+    tissue: '',
+    tumorForm: 'LIQUID',
+  },
+  {
+    id: 984,
+    subtype: '',
+    code: '',
+    color: '',
+    mainType: 'All Solid Tumors',
+    level: -1,
+    tissue: '',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 985,
+    subtype: '',
+    code: '',
+    color: '',
+    mainType: 'Germline Disposition',
+    level: -1,
+    tissue: '',
+    tumorForm: 'MIXED',
+  },
+  {
+    id: 986,
+    subtype: '',
+    code: '',
+    color: '',
+    mainType: 'Other Tumor Types',
+    level: -1,
+    tissue: '',
+    tumorForm: 'MIXED',
+  },
+  {
+    id: 987,
+    subtype: '',
+    code: '',
+    color: '',
+    mainType: 'Other Solid Tumor Types',
+    level: -1,
+    tissue: '',
+    tumorForm: 'SOLID',
+  },
+  {
+    id: 988,
+    subtype: '',
+    code: '',
+    color: '',
+    mainType: 'Other Liquid Tumor Types',
+    level: -1,
+    tissue: '',
+    tumorForm: 'LIQUID',
+  },
+];
```

Comment:
Why did we make the decision to hardcode the cancer types into the frontend?

Reply:
Thanks for catch. Thought I removed this before commit.